src/Plugin.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Sideways\Classic;
 
+use Sideways\Classic\Users\AdminUsers;
```

Comment:
do you think this should be on the skeleton instead? seems to me that we *might* want to change this for specific clients and its code that doesn't require much updates anyway, seems better just to be "copypasted" everytime we clone the skeleton

Reply:
Yep, you're right I moved this to the skeleton. Closing this PR